foreman/data_refinery_foreman/foreman/management/commands/assoc_experiment_results.py

```diff
@@ -0,0 +1,70 @@
+"""This command will go through ComputationalResult objects that are a
```

Comment:
Will we ever need to actually run this again, or can it just be done automatically as part of the migration?

Reply:
We won't need to run this again. However we can't run it at the same time as the migration because it needs the migration to have run so it can use the new data model. I could put it into a new migration, but that seemed like it'd be slightly more complicated to implement, it's not a migration we'd be able to rollback anyway, and I'm not actually sure how long this will take so I thought it might be nice to be able to do it outside of the deploy process (which is when the migration would run).